python_ta/transforms/type_inference_visitor.py

```diff
@@ -545,14 +545,13 @@ def get_attribute_class(self, t: type) -> Tuple[str, bool]:
 
         if isinstance(class_type, _ForwardRef):
             class_name = class_type.__forward_arg__
+            if class_name not in self.type_store.classes:
+                class_name = class_name.lower()
         elif hasattr(t, '__name__'):
             class_name = t.__name__
         else:
             class_name = None
 
-        if class_name not in self.type_store.classes:
```

Comment:
Hmm this change is interesting. The above `if` has three cases; I'm worried that you missed out on the middle case with this change. You could just change this condition here to `class_name is not None and ..`

Reply:
Sure thing! That was actually my original solution as well, I just felt like the if statement felt a bit long